server/src/domain/job/job.repository.ts

```diff
@@ -70,6 +71,9 @@ export type JobItem =
   // Asset Deletion
   | { name: JobName.PERSON_CLEANUP; data?: IBaseJob }
 
+  // Immich Version
+  | { name: JobName.LATEST_IMMICH_VERSION; data?: GithubRelease }
```

Comment:
data it not actually used here. You could probably just put `IBaseJob` or possibly `never`